README.md

```diff
@@ -105,6 +105,16 @@ server.setErrorConfig((app) => {
 });
 ```
 
+### `.setRoutingConfig(routingConfig)`
```

Comment:
Can you please add a note about the default value `{  rootPath: "/"  }` being used here.